client/services/helper.js

```diff
@@ -0,0 +1,63 @@
+import axios from 'axios';
+
+/**
+ *  Access JWT token from user's browser. If user has no token,
+ *  set equal to 'default' which the server will register as invalid
+ *  and redirect to login.
+ */
+var localStorageToken = local.storage.getItem('token') ? local.storage.getItem('token') : 'default';
+
+
+/**
+ *  Post helper function for making api calls to server
+ *  @expected arguments - Url, data (i.e. object of user data or product data)
+ *  @return {Object}
+ */
+var postHelper = function(url, data){
+  
+  axios.post(url, {
```

Comment:
Not quite sure what you mean, sorry!  Was it an issue with the comments or with the postHelper? @echo304